### PR TITLE
[ci] add fmt check

### PR DESCRIPTION
**Pull request by alvicsam**
Close https://github.com/paritytech/cumulus/issues/1732

### CODE REVIEW

File: .github/workflows/fmt-check.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Quick check Formatting
```
Comment (chevdor) [inline .github/workflows/fmt-check.yml:1]: I would suggest aligning the `name` and the filename.
This workflow used to be called `quick-check.yml`.

**Review by chevdor — CHANGES_REQUESTED** [review]

File: .github/workflows/fmt-check.yml (hunk 1 above)
Comment (alvicsam) [reply .github/workflows/fmt-check.yml:1]: But it's not clear from the name "quick check" what the action does. Maybe `quick-fmt-check` then?

**Review by radupopa2010 — APPROVED** [review]

File: .github/workflows/fmt-check.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+name: Rustfmt check
+
+on:
+  push:
+    branches:
+      - "*"
```
Comment (sergejparity) [inline .github/workflows/fmt-check.yml:6]: Do we really want to trigger GHA on every push to every branch?

**Review by sergejparity — APPROVED** [review]

**Review by chevdor — APPROVED** [review]

**Comment by chevdor** [comment]
bot merge

**Review by coderobe — APPROVED** [review]

**Pull request merged**